procedures/auxtel/observation_procedures/DayTime-Checkout.ipynb

```diff
@@ -0,0 +1,631 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    # Take bias and make sure it lands
</pre><p>Again, consider creating a notebook "section":</p><p> </p><pre class="ql-syntax" spellcheck="false">## Take bias and make sure it lands in the OODS
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_notebooks/pull/55/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>